.github/workflows/config/spelling_allowlist.txt

```diff
@@ -0,0 +1,78 @@
+NVIDIA
```

Comment:
Can this be sorted so it's easier to figure out what's already in the list?

Reply:
Moved to a separate PR, but yes.